curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-read-and-understand-code-documentation/66001d8b878f5141214f2d77.md

```diff
@@ -0,0 +1,93 @@
+---
+id: 66001d8b878f5141214f2d77
+title: Task 36
+challengeType: 22
+dashedName: task-36
+---
+
+<!--
+AUDIO REFERENCE:
+Entire dialogue
+-->
+
+# --description--
+
+This task involves summarizing the conversation between Tom and Sarah about understanding parameters and options in documentation. 
+
+# --fillInTheBlank--
+
+## --sentence--
+
+`Tom is _ with the _ and options in the documentation and asks how to understand them better. Sarah suggests starting with reading their _. She explains that the documentation provides details on what each parameter does and its usage. She advises looking for _ that show how to _ options correctly and the _ they have. Tom thanks Sarah for her _ and says it helps him _ through the documentation. He agrees to follow her _.`
+
+## --blanks--
+
+`confused`
+
+### --feedback--
+
+It means not being clear about something. Tom is unclear about how to understand the parameters and options.
+
+---
+
+`parameters`
+
+### --feedback--
+
+THey are special settings in functions or commands. Tom needs help understanding these.
```

Comment:
```suggestion
They are placeholder variables in the function.
```

Not sure about the suggestion, but what we have now doesn't feel right at all.

Reply:
@naomi-lgbt Thanks for catching this error. I searched other tasks to check the descriptions for "parameter", and I'll update task-28.